project4/project/ebs/templates/ebs/home.html

```diff
@@ -105,17 +96,20 @@
 	
 <!-- BANNER BLOGS POSTS -->
 
-	<div class="banner">
+	<div class="banner1">
 		<div class="container">
 			<div class="row">
 				<div class="col-md-6">
-					<div id="img1">
-						<a href="#"><img  class="banner-imgbig img-responsive " src="{% static 'ebs/img/pro1.jpg' %} " /> </a>
+					<div id="banner1-img">
+						<a href="#">
+						<img  class="banner1-imgbig img-responsive " src="{% static 'ebs/img/pro1.jpg' %} " /> 
+						</a>
 	
 					</div>
-					<div id="text1"><div class="text-block"><span class=" white-text  font1"><a href="#"><p class="banner-line">PHOTOGRAPHY</p></a></span></div>
-					<a href="#"><h3> Celebrity Make-up artist gary cockrill shows you beaty tips</h3></a>
-					<a href="#"><span class="white-text marginl font-author"><b>David Lee</b></span><span class="white-text font-author">  -  September 24,    2016</span></a> 
+					<div id="banner1-text">
+					<a href="#"><span class="catagory-block left-margin">PHOTOGRAPHY</span></a>
+					<a href="#"><h3>Celebrity Make-up artist gary cockrill shows you beaty tips</h3></a>
+					<a href="#"><span class="white-text left-margin author-font"><b>David Lee</b></span><span class="white-text font-author">  -  September 24,    2016</span></a> 
```

Comment:
avoid using multiple classes for single purpose, it will help you to reduce code.